diff -durN enemies-of-carlotta-1.0.3.orig/Makefile enemies-of-carlotta-1.0.3/Makefile
--- enemies-of-carlotta-1.0.3.orig/Makefile	Tue Feb 25 22:32:13 2003
+++ enemies-of-carlotta-1.0.3/Makefile	Sun Sep  7 17:09:25 2003
@@ -10,12 +10,12 @@
 	python testrun.py
 
 install:
-	install -d $(bindir) $(sharedir) $(man1dir) $(man1dirfr)
+	install -d $(DESTDIR){$(bindir),$(sharedir),$(man1dir),$(man1dirfr)}
 	sed 's,^SHAREDIR=.*,SHAREDIR="$(sharedir)",' enemies-of-carlotta \
-	    > $(bindir)/enemies-of-carlotta
-	chmod 755 $(bindir)/enemies-of-carlotta
-	sh fix-config $(sharedir) < eoc.py > $(sharedir)/eoc.py
-	python -c 'import py_compile; py_compile.compile("$(sharedir)/eoc.py")'
-	install -m 0644 enemies-of-carlotta.1 $(man1dir)
-	install -m 0644 enemies-of-carlotta.1.fr $(man1dirfr)/enemies-of-carlotta.1
-	install -m 0644 templates/????* $(sharedir)
+	    > $(DESTDIR)$(bindir)/enemies-of-carlotta
+	chmod 755 $(DESTDIR)$(bindir)/enemies-of-carlotta
+	sh fix-config $(sharedir) < eoc.py > $(DESTDIR)$(sharedir)/eoc.py
+	python -c 'import py_compile; py_compile.compile("$(DESTDIR)$(sharedir)/eoc.py")'
+	install -m 0644 enemies-of-carlotta.1 $(DESTDIR)$(man1dir)
+	install -m 0644 enemies-of-carlotta.1.fr $(DESTDIR)$(man1dirfr)/enemies-of-carlotta.1
+	install -m 0644 templates/????* $(DESTDIR)$(sharedir)
